osu.Game/Screens/OnlinePlay/Multiplayer/Match/MultiplayerReadyButton.cs

```diff
@@ -105,6 +107,13 @@ private void updateState()
 
             button.Enabled.Value = Client.Room?.State == MultiplayerRoomState.Open && !operationInProgress.Value;
 
+            // When the local user is the host and spectating the match, the "start match" state should be enabled.
+            if (localUser.State == MultiplayerUserState.Spectating)
+            {
+                button.Enabled.Value &= Room?.Host?.Equals(localUser) == true;
+                button.Enabled.Value &= newCountReady > 0;
+            }
+
```

Comment:
Not sure about doing this in multiple steps... means the button could enter enabled and then disabled in the same call to this method. Probably best extracting a bool out and setting it to `button.Enabled.Value` at the end?